net-libs/libwebsockets/libwebsockets-4.0.1-r1.ebuild

```diff
@@ -47,20 +47,26 @@ src_configure() {
 	local mycmakeargs=(
 		-DCMAKE_DISABLE_FIND_PACKAGE_Git=ON
 		-DLWS_HAVE_LIBCAP=$(usex caps)
+		-LWS_HAVE_SYS_CAPABILITY_H=$(usex sys-capability)
 		-DLWS_IPV6=$(usex ipv6)
 		-DLWS_ROLE_DBUS=$(usex dbus)
+		-DLWS_SSL_CLIENT_USE_OS_CA_CERTS=$(usex ssl_ca_certs)
 		-DLWS_WITHOUT_CLIENT=$(usex !client)
 		-DLWS_WITHOUT_TEST_CLIENT=$(usex !client)
 		-DLWS_WITH_ACCESS_LOG=$(usex access-log)
 		-DLWS_WITH_CGI=$(usex cgi)
+		-DLWS_WITH_DIR=$(usex dir)
+		-DLWS_WITH_FILE_OPS=$(usex file-ops)
 		-DLWS_WITH_GENERIC_SESSIONS=$(usex generic-sessions)
 		-DLWS_WITH_HTTP2=$(usex http2)
 		-DLWS_WITH_HTTP_PROXY=$(usex http-proxy)
+		-DLWS_WITH_HTTP_UNCOMMON_HEADERS=$(usex http-uncommon-headers)
```

Comment:
Is there a reason to make this optional?  The fact that you're using mbedtls makes me thing you're doing this for embedded work - in which case, yes it makes sense to be able to strip out as much as possible.

Reply:
Yes, this is for embedded work, hence these options